requirements.in

```
@@ -35,12 +35,12 @@ luqum==0.13.0
 mapbox==0.18.1
 markdown==3.4.1
 networkx==3.2.1
-numpy==1.24.3
 oauthlib
 openpyxl==3.1.0  # for reading .xlsx files (but not .xls)
-Pillow==9.5.0
+numpy==1.26.1
+Pillow==10.1.0
 pip-tools==7.1.0
-psycopg2-binary==2.9.6
+psycopg2-binary==2.9.9
 PyJWT==2.6.0
 pyparsing==2.4.7
 pysndfile==1.4.4
```

I needed to upgrade this
```suggestion
pysndfile==1.4.7
```

---

locally on macos (that is)

---

we discussed this and we'll stay with the old version, which works on our docker setup